### PR TITLE
refacto: spec types

### PR DESCRIPTION
**Pull request by muchafm**

### CODE REVIEW

**Review by Zusoy — APPROVED** [review]

File: tests/Gaufrette/Functional/Adapter/AzureBlobStorageTest.php [hunk 1]
```diff
@@ -13,10 +13,10 @@
 class AzureBlobStorageTest extends FunctionalTestCase
 {
     /** @var string Name of the Azure container used */
-    private $container;
+    private string $container;
 
     /** @var AzureBlobStorage */
```
Comment (KevinArtus) [inline tests/Gaufrette/Functional/Adapter/AzureBlobStorageTest.php:18]: To remove

File: tests/Gaufrette/Functional/Adapter/AzureBlobStorageTest.php [hunk 2]
```diff
@@ -13,10 +13,10 @@
 class AzureBlobStorageTest extends FunctionalTestCase
 {
     /** @var string Name of the Azure container used */
```
Comment (KevinArtus) [inline tests/Gaufrette/Functional/Adapter/AzureBlobStorageTest.php:15]:
```suggestion
    /** Name of the Azure container used */
```

**Review by KevinArtus — CHANGES_REQUESTED** [review]

File: spec/Gaufrette/Adapter/functions.php [hunk 3]
```diff
@@ -30,17 +30,17 @@ function extension_loaded($name)
     return $extensionLoaded;
 }
 
-function opendir($url)
+function opendir()
 {
     return true;
 }
 
-function apc_fetch($path)
```
Comment (PedroTroller) [inline spec/Gaufrette/Adapter/functions.php:38]: Same: http://php.adamharvey.name/manual/en/function.apc-fetch.php

File: spec/Gaufrette/Adapter/functions.php [hunk 4]
```diff
@@ -30,17 +30,17 @@ function extension_loaded($name)
...
+function apc_fetch(string $path)
 {
     return sprintf('%s content', $path);
 }
 
-function apc_store($path, $content, $ttl)
```
Comment (PedroTroller) [inline spec/Gaufrette/Adapter/functions.php:43]: Same: http://php.adamharvey.name/manual/en/function.apc-store.php

File: spec/Gaufrette/Adapter/functions.php [hunk 5]
```diff
@@ -58,7 +58,7 @@ function apc_delete($path)
     return true;
 }
 
-function apc_exists($path)
```
Comment (PedroTroller) [inline spec/Gaufrette/Adapter/functions.php:61]: Same: http://php.adamharvey.name/manual/en/function.apc-exists.php

**Review by KevinArtus — APPROVED** [review]

**Review by PedroTroller — CHANGES_REQUESTED** [review]

**Review by PedroTroller — APPROVED** [review]

**Pull request closed without merging**